golem_messages/message.py

```diff
@@ -1358,6 +1358,39 @@ def deserialize_slot(self, key, value):
         return deserialize_task_to_compute(key, value)
 
 
+class FileTransferToken(Message):
+    TYPE = CONCENT_MSG_BASE + 5
+
+    __slots__ = [
+        'token_expiration_deadline',
+        'storage_cluster_address',
+        'authorized_client_public_key',
+        'files',
+        'operation',
+        'file_sizes',
+        'md5_hashes',

```

Comment:
What is the format of `files`, `file_sizes` and `md5_hashes`? Can they be combined into single list of dicts (or `datastructures.FrozenDict` if needed):

```python
files_data = [
    {
        'filepath': '/a/b/c/file0001',
        'checksum': 'sha1:XXXXX',
        'size': 1024,
    },
]

Reply:
okey, I've change it as You suggested with:
```
class TokenFilesData(datastructures.FrozenDict):
    """Represents SUBTASK metadata."""
    ITEMS = {
        'filepath'  :'',
        'md5_hashes':'',
        'size'      :0,
    }
```